data_store/table_extraction/extract.py

```diff
@@ -72,10 +72,10 @@ def extract(self, worksheet_name: str, id_tag: str) -> list[Table]:
     def _get_tags(self, id_tag: str, worksheet: pd.DataFrame) -> tuple[list[Cell], list[Cell]]:
         start_tag = self.START_TAG.format(id=id_tag)
         end_tag = self.END_TAG.format(id=id_tag)
-        start_tags = list(zip(*np.where(worksheet == start_tag), strict=False))
-        end_tags = list(zip(*np.where(worksheet == end_tag), strict=False))
-        start_tags = [Cell(row, col) for row, col in start_tags]
-        end_tags = [Cell(row, col) for row, col in end_tags]
```

Comment:
Reassigning `start_tags` and `end_tags`  upsets mypy.

Reply:
```suggestion
        filtered_start_tags = zip(*np.where(worksheet == start_tag), strict=False)
        filtered_end_tags = zip(*np.where(worksheet == end_tag), strict=False)
```

nit: Don't need to turn them into lists if they're only going to be used once to build the lists later, they could stay as generators